### PR TITLE
ZER-110/Contact us from

### PR DESCRIPTION
**Pull request by anko20094**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: app/controllers/admins/messages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+module Admins
+  class MessagesController < BaseController
+    
+  def index
+    @message = Message.all.order created_at: :desc
```
Comment (olegsobchuk) [inline app/controllers/admins/messages_controller.rb:5]: remove `all` and wrap parameters to parentheses

File: app/controllers/messages_controller.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+class MessagesController < ApplicationController
+
+  def new
+    @message = Message.new
+  end
+
+  def index
+    @message = Message.all.order created_at: :desc
```
Comment (olegsobchuk) [inline app/controllers/messages_controller.rb:8]: remove all and wrap parameters to parentheses

File: app/controllers/messages_controller.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  end
+
+  def create
+    @message = Message.new(message_params)
+    if @message.save
+      redirect_to root_path
```
Comment (olegsobchuk) [inline app/controllers/messages_controller.rb:14]: redirect with notification

File: app/controllers/messages_controller.rb (hunk 2 above)
Comment (anko20094) [reply app/controllers/messages_controller.rb:8]: what does it mean?

File: app/controllers/admins/messages_controller.rb (hunk 1 above)
Comment (anko20094) [reply app/controllers/admins/messages_controller.rb:5]: what does it mean?

**Review by olegsobchuk — APPROVED** [review]

**Review by aleksasha-o — APPROVED** [review]

**Pull request merged**